app/components/Settings/Scheduler/previewModal.jsx

```diff
@@ -0,0 +1,248 @@
+import React, {Component} from 'react';
+import PropTypes from 'prop-types';
+import { Controlled as CodeMirror } from 'react-codemirror2';
+import ms from 'ms';
+
+import Modal from '../../modal.jsx';
+import { Link } from '../../Link.react.js';
+import { FrequencySelector } from './createModal.jsx';
+import { Row, Column } from '../../layout.jsx';
+import SQL from './sql.jsx';
+import { plotlyUrl } from '../../../utils/utils.js';
+import { mapDialect } from './util.js';
+
+const NO_OP = () => {};
+
+const rowStyle = {
+    justifyContent: 'flex-start',
+    borderBottom: '1px solid rgba(0, 0, 0, 0.05)',
+    padding: '16px 0px'
+};
+const boxStyle = { boxSizing: 'border-box', width: '50%' };
+
+export class PreviewModal extends Component {
```

Comment:
Add description; e.g.: "*implements a modal window to view details of a scheduled query*"